cloud-tasks/app/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "postcard-app",
+  "version": "1.0.0",
+  "description": "Sends postcard when triggered by a Cloud Task",
+  "main": "index.js",
+  "license": "Apache-2.0",
+  "author": "Google LLC",
+  "scripts": {
+    "start": "node index.js",
+    "test": "mocha",
+    "lint": "eslint '**/*.js'"
```

Comment:
This inspired me to look up https://eslint.org/docs/2.0.0/user-guide/configuring#configuration-file-formats, where I learned eslint will traverse upwards for configuration. Neat!